### ISSUE TITLE
U+3388 and U+3389: “cal” is displayed in italic

### ISSUE THREAD

**Issue by ghost** [adobe-fonts/source-han-serif#223]
### Description

U+3388 and U+3389 are shown with an italic “cal”. This unit symbol should not be italicized in the regular style (for comparison: [NIST](https://www.nist.gov/pml/special-publication-811/nist-guide-si-appendix-b-conversion-factors/nist-guide-si-appendix-b8#C), [SI](https://www.bipm.org/documents/20126/41483022/SI-Brochure-9-EN.pdf#page=33)).

**Comment by prakashDiana** [comment]
I don't think this is an issue. In some fonts, the "cal" symbol can be italicized, like "㎈" and "㎉".
Non-italicized: MS Gothic, Source Han Sans
Italicized: Source Han Serif, YuMincho (l (lovercase L) is curved)

**Comment by ChinaMrAlvin** [comment]
> I don't think this is an issue. In some fonts, the "cal" symbol can be italicized, like "㎈" and "㎉". Non-italicized: MS Gothic, Source Han Sans Italicized: Source Han Serif, YuMincho (l (lovercase L) is curved)
